src/index.js

```diff
@@ -92,6 +92,10 @@ function radredis(schema, transforms, port, host, options){
         .map((objs) => { this.push(objs) })
         .then(() => callback() )
       }))
+    },
+
+    count: () => {
+      return redis.get(`${keyspace}:id`).then(count => parseInt(count, 10))
```

Comment:
Would also be nice to support counting other indexes. 

`count('scheduled_at') => all scheduled posts`
`count('type', 4) => all posts with type 4`

I'm cool with not implementing this right now, but if not please file an issue for it for later.